templates/ubuntu.sh

```diff
@@ -33,23 +33,29 @@ chmod +x /usr/local/bin/rds_restore_cluster_from_snapshot.sh
 cat <<"__EOF__" > /usr/local/include/Makefile.${name}.mysql
 DUMP ?= /tmp/mysqldump.sql
 
-.PNONY : ${name}\:db-import
-DB ?= app
+.PHONY : ${name}\:db-import
+DB ?= ${db_name}
 ## Import dump
 ${name}\:db-import:
 	$(eval MY_CNF?=/root/${name}.my.cnf)
 	@pv $(DUMP) | sudo mysql --defaults-file=$(MY_CNF)
 	MY_CNF=$(MY_CNF) DB=$(DB) /usr/local/bin/mysql_latin_utf8.sh | pv | sudo mysql --defaults-file=$(MY_CNF) $(DB)
+
+## DB connect
+${name}\:db-connect:
+	$(eval MY_CNF?=/root/${name}.my.cnf)
```

Comment:
It might not work as expected. `MY_CNF` may have empty value.
![screenshot from 2017-11-10 10-58-02](https://user-images.githubusercontent.com/11299538/32650329-0b04f482-c606-11e7-99bb-ebf9b7910b6c.png)

Reply:
There is requirement for my.cnf format that we support now.